compile/unminified/pm-image-editor.js

```diff
@@ -198,10 +198,11 @@
             },
             link: function (scope, element) {
                 var buttonNames = 'crop,rotate-cw,rotate-acw,flip-h,flip-v,undo,redo';
+                var buttonLabels = ['Crop', 'Rotate Clockwise', 'Rotate Counterclockwise', 'Flip Horizontal', 'Flip Vertical', 'Undo', 'Redo'];
```

Comment:
I don't think we can do that in this library as it's open sourced, maybe we can create a decorator